### PR TITLE
 update change_log

### PR DESCRIPTION
**Pull request by cyjseagull**
pls refer to:  

https://cyjseagull-doc.readthedocs.io/zh_CN/release-2.0-new

### CODE REVIEW

File: docs/change_log/2_0_0_rc1.md [hunk 1]
```diff
@@ -3,9 +3,11 @@
 ```eval_rst
 .. admonition:: v1.x升级到v2.0.0-rc1
 
-    - **v2.0.0-rc2不兼容v1.x，v2.0.0-rc1无法直接解析v1.x产生的历史区块数据** ，但可通过在v2.0.0-rc2的新链上执行历史交易的方式恢复旧数据
+    - **v2.0.0-rc2不兼容v1.x，v2.0.0-rc1无法直接解析v1.x产生的历史区块数据** ，但可通过在 `v2.0.0-rc1 <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v2.0.0-rc1/fisco-bcos.tar.gz>`_ 的新链上执行历史交易的方式恢复旧数据
 
     - **搭建2.0的新链** ：请参考 `安装 <../installation.html>`_ 
+
+    - `v2.0.0-rc1 release <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/tag/v2.0.0-rc1>`_
```
Comment (qyan-dev) [inline docs/change_log/2_0_0_rc1.md]: v2.0.0-rc1 release notes

File: docs/change_log/2_0_0_rc2.md [hunk 2]
```diff
@@ -3,9 +3,11 @@
 ```eval_rst
 .. admonition:: v2.0.0-rc1升级到v2.0.0-rc2
 
-    - **兼容升级** ：直接替换v2.0.0-rc1节点的二进制为v2.0.0-rc2二进制，升级后的版本修复v2.0.0-rc1中的bug，但不会启用v2.0.0-rc2并行计算、分布式存储等新特性， **升级到v2.0.0-rc2后，无法回滚到v2.0.0-rc1**
+    - **兼容升级** ：直接替换v2.0.0-rc1节点的二进制为 `v2.0.0-rc2二进制 <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v2.0.0-rc2/fisco-bcos.tar.gz>`_ ，升级后的版本修复v2.0.0-rc1中的bug，但不会启用v2.0.0-rc2并行计算、分布式存储等新特性， **升级到v2.0.0-rc2后，无法回滚到v2.0.0-rc1**
 
     - **全面升级** ：参考 `安装 <../installation.html>`_ 搭建新链，重新向新节点提交所有历史交易，升级后节点包含v2.0.0-rc2新特性
+
+    - `v2.0.0-rc2 release <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/tag/v2.0.0-rc2>`_
```
Comment (qyan-dev) [inline docs/change_log/2_0_0_rc2.md]: v2.0.0-rc2 release notes

File: docs/change_log/2_0_0_rc1.md (hunk 1 above)
Comment (cyjseagull) [reply docs/change_log/2_0_0_rc1.md]: already modified from `release note` to `Relase Note`.

File: docs/change_log/2_0_0_rc2.md (hunk 2 above)
Comment (cyjseagull) [reply docs/change_log/2_0_0_rc2.md]: already modified from `release note` to `Relase Note`

**Pull request merged**